src/main/java/uk/gov/companieshouse/controller/ExceptionHandlerController.java

```diff
@@ -60,7 +57,7 @@ public void handleNotFound(RuntimeException ex, HttpServletRequest request) {
     }
 
     @ResponseStatus(HttpStatus.BAD_REQUEST)
-    @ExceptionHandler(DirectorNotPendingApprovalException.class)
+    @ExceptionHandler({DirectorNotPendingApprovalException.class, DissolutionApplicationWrongStatusException.class})
     public void handleDirectorNotPendingApproval(RuntimeException ex, HttpServletRequest request) {
```

Comment:
Hmmmm, a couple of things here:
1. The method name no longer reflects the exceptions it handles
2. This is a global exception handler and should be kept pretty generic imo. This is beginning to look like it is treated as the **only** exception handler in the application, which shouldn't be the case as it will scale terribly. In my opinion the individual controllers should handle all service exceptions and map them to a HTTP Status Code exception i.e.:
```
try {
  return dissolutionService.addDirectorApproval(companyNumber, userId, request.getRemoteAddr(), body.getEmail());
} catch (DissolutionApplicationWrongStatusException ex) {
  throw new BadRequestException();
}
```

You should be able to look at a controller and understand the entry and exit points of that endpoint, including all status codes.